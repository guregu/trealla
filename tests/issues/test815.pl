:- initialization(main).

main :-
	L="0"||_,
	format("~q~n",[L]),
	L0="0"||abc,
	format("~q~n",[L0]),
	L1="0"||123,
	format("~q~n",[L1]).
