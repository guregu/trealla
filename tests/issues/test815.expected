"0"||_0
"0"||abc
"0"||123
